package.json

```diff
@@ -63,23 +63,25 @@
     "jasmine-node": "^3.0.0",
     "karma": "^4.4.1",
     "karma-chrome-launcher": "^3.1.0",
+    "karma-cli": "^2.0.0",
     "karma-coverage": "^2.0.1",
-    "karma-es6-shim": "^1.0.0",
-    "karma-jasmine": "2.0.1",
-    "karma-phantomjs-launcher": "1.0.4",
+    "karma-firefox-launcher": "1.0.0",
+    "karma-jasmine": "^3.1.0",
+    "karma-phantomjs-launcher": "^1.0.4",
     "karma-safari-launcher": "1.0.0",
     "karma-webpack": "^4.0.2",
     "matchdep": "^2.0.0",
-    "request": "2.88.0",
+    "request": "^2.88.0",
     "sinon": "^7.5.0",
     "standard": "^14.3.1",
-    "uglifyjs-webpack-plugin": "^2.2.0",
-    "webpack": "^4.41.2",
-    "webpack-cli": "^3.3.10",
-    "karma-cli": "^2.0.0"
+    "webpack": "^4.41.5",
+    "webpack-cli": "^3.3.10"
   },
   "dependencies": {
-    "http-https": "^1.0.0"
+    "http-https": "^1.0.0",
+    "karma-es6-shim": "^1.0.0",
```

Comment:
Not sure if you are still interested in doing this but could you move the this new deps to devDependencies?